.github/workflows/continuous-integration.yml

```diff
@@ -19,12 +19,9 @@ jobs:
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
     steps:
       - uses: actions/checkout@v1
-      - name: Set Ruby version to use
-        run: |
-          echo "::set-env name=RUBY_VERSION::$(cat .ruby-version)"
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: ${{ env.RUBY_VERSION }}
+          ruby-version: 2.6.3
```

Comment:
You don't need this. The `ruby/setup-ruby` action reads the `.ruby-version`.

Reply:
I did wonder!!!